### ISSUE TITLE
"it doesn't burn more electricity than small nations"

### ISSUE THREAD

**Issue by infominer33** [keybase/keybase-issues#3400]
I'm really falling in love with Keybase, want to promote it's use and see its success for years to come.

However It was raised to my attention today about some of the copy in the stellar wallet setup.

![image](https://user-images.githubusercontent.com/44134283/56935903-31032480-6ac2-11e9-946c-bbe07cb3e9b9.png)

This jab at PoW is obvious stellar rhetoric, and is disingenuous, as you use the Bitcoin blockchain as an anchor, *NOT* stellar.

Removing that line from the wallet sign-up would be a great PR move.

[![image](https://user-images.githubusercontent.com/44134283/56935984-b090f380-6ac2-11e9-877a-3dae0b29bd2d.png)](https://twitter.com/michaelfolkson/status/1122476445035638784)
https://twitter.com/michaelfolkson/status/1122476445035638784

**Comment by Laurelai** [comment]
But its factually true. Keybase *doesnt* burn more electricity than small nations.

**Comment by infominer33** [comment]
yes, and public relations is all about conveying precise factual truth

**Comment by junderw** [comment]
Neither does Bitcoin.

The implied jab is all based on biased studies, and no one compares the electricity usage to the entire banking sector.

If we want to take implicit jabs here I've got one.

"Well, at least Bitcoin is not a security as defined by the SEC."

oh, what's that you say? "How dare I imply that a centralized coin be regulated as a security!" ?

why my graces I have implied no such thing my darlings!

Seems like I struck a nerve there, but for the record I have never explicitly stated XLM is a security.

Yeah, this post is of bad taste, just like the jab.

Come on.

**Comment by Laurelai** [comment]
It doesnt bother me, regulating finance is good, actually.

**Comment by infominer33** [comment]
besides the point of why keybase would make itself harder to promote among one of the most likely userbases to sign up.

**Comment by Laurelai** [comment]
Mabey they deliberately dont want to be associated with bitcoin. Have you considered this a possibility?

**Comment by Laurelai** [comment]
Or mabey its just a joke and you need to lighten up

**Comment by dmbr0** [comment]
How much energy does the banking sector consume with the gas it takes to get its employees to work every day? Building energy consumption? The energy embodied in building cars to get the people to the buildings? The energy consumed by data centers? How about the energy that isn't captured and converted to work on a daily basis? It's incomprehensible. I don't really mind that stellar is in the application but can we at least get some bitcoin tools?

**Comment by junderw** [comment]
Fixed in https://github.com/keybase/client/pull/17504

**Comment by junderw** [comment]
https://twitter.com/ofnumbers/status/1128866873318223872?s=09

Looks like Stellar should maybe use a little more electricity... :-/
